kloppy/infra/serializers/event/wyscout/deserializer_v3.py

```diff
@@ -347,6 +344,12 @@ def deserialize(self, inputs: WyscoutInputs) -> EventDataset:
                     "period": periods[-1],
                     "timestamp": float(
                         raw_event["second"] + raw_event["minute"] * 60
+                    )
+                    if period_id == 1
+                    else float(
```

Comment:
How about matches that go into extra time or penalty shoot-outs?

Reply:
Wyscout has an example in the API documentation. They distinguish between five different periods.

Currently, the deserializer would struggle with matches that go into extra time already due to how the `matchPeriod` field is handled. Hence, I'm fine with keeping your current approach.

```json
{
  "id": 533182244,
  "matchId": 2050345,
  "matchPeriod": "P",
  "minute": 125,
  "second": 45,
  "matchTimestamp": "02:05:45.456",
  "videoTimestamp": "8760.456735",
  "relatedEventId": null,
  "type": {
    "primary": "postmatch_penalty",
    "secondary": []
  },
  "location": {
    "x": 90,
    "y": 50
  },
  "team": {
    "id": 1841,
    "name": "Zenit",
    "formation": "4-1-3-2"
  },
  "opponentTeam": {
    "id": 1843,
    "name": "Lokomotiv Moskva",
    "formation": "4-4-1-1"
  },
  "player": {
    "id": 17686,
    "name": "D. Criscito",
    "position": "LB"
  },
  "pass": null,
  "shot": {
    "bodyPart": "left_foot",
    "isGoal": true,
    "onTarget": true,
    "goalZone": "gbr",
    "xg": null,
    "postShotXg": null,
    "goalkeeperActionId": 533182178,
    "goalkeeper": {
      "id": 8992,
      "name": "Guilherme",
      "reflexSave": false
    }
  },
  "groundDuel": null,
  "aerialDuel": null,
  "infraction": null,
  "carry": null,
  "possession": null
}
```